### PR TITLE
calcurse: update to 4.8.1

### PR DESCRIPTION
**Pull request by contextnerror**
Closes: https://trac.macports.org/ticket/65451

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.6 18G9323 x86_64
Command Line Tools 10.3.0.0.1.1562985497

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by mklein-de** [comment]
looks good, thx!

File: sysutils/calcurse/Portfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PortSystem 1.0
```
Comment (herbygillot) [inline sysutils/calcurse/Portfile:1]: @contextnerror, can you add a modeline?  The first line of any Portfile should ideally be the modeline (ref: https://guide.macports.org/#development.creating-portfile )

```suggestion
# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
PortSystem 1.0
```

File: sysutils/calcurse/Portfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
 name            calcurse
-version         4.5.1
+version         4.8.1
 categories      sysutils
 platforms       darwin
```
Comment (herbygillot) [inline sysutils/calcurse/Portfile:6]: `platforms darwin` is now implicit, and this line can be removed from Portfiles.

**Comment by herbygillot** [comment]
Alright, I've squashed from my side.

In the future, you can squash multiple commits for a PR by doing an interactive rebase (in which you mark the extra commits as `squash`) which consolidates everything back into the single original commit, and then force-pushing onto the PR's remote branch.

**Pull request merged**